src/commands/everyone.js

```diff
@@ -0,0 +1,53 @@
+const { chattools } = require('../utils');
+
+const STRINGS = {
+  help: `
+Marca todos os membros do grupo com a mensagem citada.
+
+*uso:* \`\`\`!everyone [--args]\`\`\`
+
+*args válidos:*
+  \`\`\`--help\`\`\` -> _mostra esta mensagem._
+  `.trim(),
+  defaultMessage: '@everyone',
+};
+
+class Everyone {
+  constructor() {
+    this.name = 'everyone';
+    this.strings = STRINGS;
+  }
+
+  async execute(data, message) {
+    const { args } = data;
+    const isAdm = await chattools.isAdm(message);
+
+    if (!isAdm) {
+      message.reply('staff only.');
+      return;
+    }
+
+    if (args.includes('help')) {
+      message.reply(this.strings.help);
+      return;
+    }
+
+    const chat = await message.getChat();
+    const { participants } = chat;
+
+    if (message.hasQuotedMsg) {
+      const quotedMessage = await message.getQuotedMessage();
+
+      chat.sendMessage(this.strings.defaultMessage, {
+        // eslint-disable-next-line no-underscore-dangle
+        quotedMessageId: quotedMessage.id._serialized,
+        mentions: participants,
+      });
```

Comment:
Quase lá, Emerson! Só mais uma mudança e o PR será aprovado.

Sempre que possível, para manter o código mais limpo, use `Message.reply` ao invés de `Chat.sendMessage`. O erro que você recebeu ao tentar usar o método `Message.reply` ocorreu porque você estava passando o param `options` no lugar do param `chatId`, para resolver isso é só passar `undefined` no lugar do `chatId`. Os parametros do método podem ser vistos [aqui](https://docs.wwebjs.dev/Message.html#reply)
```suggestion
      quotedMessage.reply(this.strings.defaultMessage, undefined, {
        mentions: participants,
      });
```

Reply:
Ahh, entendi.